### ISSUE TITLE
cannot find the stdbool.h

### ISSUE THREAD

**Issue by Simonhong111** [MrGF/py-faster-rcnn-windows#3]
hello, thanks for your code  and i am trying to run your code ,however,when i complie the lib,i found that i met the error that the pyhon had no Miscrosoft Visual c++9.0,then i install the package by the guide ,but i met another error that python couldn't find the **_# stdbool.h_**,3Q for your help
-

**Comment by Simonhong111** [comment]
it will be better you give a more details of the guide

**Comment by xjock** [comment]
Have you solved the problem?

**Comment by Simonhong111** [comment]
Yes ，I have. you can use  vs 2013 ,however you should set your  VS90COMNTOOLS  your own address like E:\Program Files\Microsoft Visual Studio 12.0\Common7\Tools\    then your complier will use vs 2013 not vs 2008

**Comment by IronDroid2088** [comment]
Could you tell me how to set my  "VS90COMNTOO"? I have no idea what it is. Thank YOU!!

**Comment by OneDirection9** [comment]
I try to set VS90COMNTOOLS, but it doesn't work. What can i do next.

**Comment by Vandertic** [comment]
You saved me. I've been banging my head without finding a solution before.

**Comment by Jumabek** [comment]
It didn;t work for me.
I had the same problem as @Simonhong111 
however pointing VS90COMNTOOLS to C:\Program Files x86\Microsoft Visual Studio 12.0\Common7\Tools\ did not solve the issue.

**Comment by zhonhel** [comment]
I solved it.
What I said maybe amazing but it really works to me.

(1)     Install VCForPython27.msi
(2)       Add `VS90COMNTOOLS` env var, and assign it `D:\vs2015\Common7\Tools\` (maybe other version VS is also ok)
(3)       Run VCForPython27.msi again, choose remove.

Now run  `python setup.py install`.  It won't appear cannot find stdbool.h.

**Comment by cgvict** [comment]
@zhonhel This method is worked for me.

**Comment by anandkoirala** [comment]
@zhonhel also worked for me for setup.py
and surprisingly  setup_cuda.py also installed which had errors before this fix

**Comment by ray2004a** [comment]
@zhonhel  thanks for help, it really works for me!! After remove "VCForPython27.msi", the error was gone =D

**Comment by AvaniPitre** [comment]
I am having same issue, followed steps by @zhonhel but, after I remove VCForPython27.msi, I get error for VS 2009 not found,  Microsoft Visual C++ 9.0 is required. Get it from http://aka.ms/vcpython27
For env variable VS90COMNTOOLS  I set vs2010 path.

**Comment by zoufangyu1987** [comment]
E:\06-prj\DOD\caffe_2013\py-faster-rcnn-master\lib>python setup.py install
home = C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v7.5

running install
running bdist_egg
running egg_info
writing fast_rcnn.egg-info\PKG-INFO
writing top-level names to fast_rcnn.egg-info\top_level.txt
writing dependency_links to fast_rcnn.egg-info\dependency_links.txt
reading manifest file 'fast_rcnn.egg-info\SOURCES.txt'
writing manifest file 'fast_rcnn.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_ext
skipping 'utils\bbox.c' Cython extension (up-to-date)
skipping 'nms\cpu_nms.c' Cython extension (up-to-date)
skipping 'pycocotools\_mask.c' Cython extension (up-to-date)
building 'pycocotools._mask' extension
C:\Users\test\AppData\Local\Programs\Common\Microsoft\Visual C++ for Python\9.0\VC\Bin\amd64\cl.exe /c /nologo /Ox /MD /
W3 /GS- /DNDEBUG -IC:\Users\test\Anaconda2\lib\site-packages\numpy\core\include -Ipycocotools -IC:\Users\test\Anaconda2\
include -IC:\Users\test\Anaconda2\PC /Tcpycocotools\maskApi.c /Fobuild\temp.win-amd64-2.7\Release\pycocotools\maskApi.ob
j -std=c99
cl : Command line warning D9002 : ignoring unknown option '-std=c99'
maskApi.c
e:\06-prj\dod\caffe_2013\py-faster-rcnn-master\lib\pycocotools\maskApi.h(8) : fatal error C1083: Cannot open include fil
e: 'stdbool.h': No such file or directory
error: command 'C:\\Users\\test\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\amd64
\\cl.exe' failed with exit status 2

Who can help me ? I tried a lot of ways.

**Comment by zoufangyu1987** [comment]
@Simonhong111  @zhonhel  @Vandertic

**Comment by zoufangyu1987** [comment]
I've done it, amending it as follows:
find the file: 
.\Anaconda2\Lib\distutils\msvc9compiler.py

Locate to line 304: VERSION = get_build_version()
replace it with : VERSION = 12.0

**Comment by Maulshree27** [comment]
Hi @zhonhel ,
I am unable to add vc90comntools env var. Could you please provide me steps to do that.
